src/test/java/core/basesyntax/service/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.List;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static final FruitTransactionParser parser = new FruitTransactionParserImpl();
+    private static final int FIRST_TRANSACTION_INDEX = 0;
+
+    @Test
+    public void parseTransaction_Ok() {
+        List<String> listToParse = List.of("type,fruit,quantity",
+                "b,banana,20");
+        FruitTransaction expectedTransaction = new FruitTransaction(
+                FruitTransaction.Operation.getOperation("b"),
+                new Fruit("banana"),
+                20);
+        assertEquals(expectedTransaction,
+                parser.parseTransaction(listToParse).get(FIRST_TRANSACTION_INDEX));
```

Comment:
it would be better to check that lists are equal instead of check only first element

Reply:
list wil not be equal, because list which came from file contains header, and header should be ignored while parsing. So i compare element of list before and after parsing. To copare correctly i should skip first element i list from file which is header line

Reply:
I meant that we can write `assertEquals(*expectedTransactions*, parser.parseTransaction(listToParse))`
or you can check size of returned list